phpunit.xml.dist

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./Tests/bootstrap.php" colors="true">
+<phpunit bootstrap="vendor/autoload.php" colors="true">
+
+    <php>
+        <const name="PASSWORD_BCRYPT" value="1" />
```

Comment:
Why that ? I don't think it makes sense at all to try to redefine a built-in constant.